### PR TITLE
#Dp 32 : UX : Make it clear that it Ok to contribute to only part of Texbooks

### PR DESCRIPTION
**Pull request by Nilesh5995**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 1]
```diff
@@ -90,6 +90,9 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{programContext.name}} </h5
         <div *ngIf="selectedCollectionIds.length > 0">
           <button class="sb-btn sb-btn-normal sb-btn-primary disabled {{selectedCollectionIds.length > 0 ? 'active' : 'disabled'}}"
             (click)="showContentTypeModal= true;">{{resourceService.frmelmnts.btn.nominate}}</button>
+            <div class="py-10">
+              <p>Note : Contribution need not be for all textbooks and all contents.You can choose to contribute as much as you want.</p>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:94]: @Nilesh5995 lang constant

File: src/app/resourcebundles/data/consumption/en.properties [hunk 2]
```diff
@@ -1277,6 +1277,7 @@ frmelmnts.lbl.inviteUser=Invite User
 frmelmnts.lbl.profileDetails=Profile Details
 frmelmnts.lbl.contributionReview=Contribution & Review
 frmelmnts.btn.nominate=Nominate
+frmelmnts.lbl.nominationClickUi=Note : Contribution need not be for all \n textbooks and all contents.You can choose \n to contribute as much as \n you want.
```
Comment (twsvaishali) [inline src/app/resourcebundles/data/consumption/en.properties:1280]: change the name to nominationMsg .. why \n is given in between? And also remove css

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 3]
```diff
@@ -90,8 +90,8 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{programContext.name}} </h5
         <div *ngIf="selectedCollectionIds.length > 0">
           <button class="sb-btn sb-btn-normal sb-btn-primary disabled {{selectedCollectionIds.length > 0 ? 'active' : 'disabled'}}"
             (click)="showContentTypeModal= true;">{{resourceService.frmelmnts.btn.nominate}}</button>
-            <div class="py-10">
-              <p>Note : Contribution need not be for all textbooks and all contents.You can choose to contribute as much as you want.</p>
+            <div class="py-10 nominateMessage">
+              {{resourceService.frmelmnts.lbl.nominationClickUi}}
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html]: where in en.properties file?

File: src/app/resourcebundles/data/consumption/en.properties (hunk 2 above)
Comment (Nilesh5995) [reply src/app/resourcebundles/data/consumption/en.properties:1280]: /n is required for new line, and css is also required for create the UI suggested by kamesh.
UI link

Comment (twsvaishali) [reply src/app/resourcebundles/data/consumption/en.properties:1280]: \n, that will be decided on the basis of how much space we have. Ekta will fix that with css.

Comment (Nilesh5995) [reply src/app/resourcebundles/data/consumption/en.properties:1280]: ok

**Pull request merged**